src/entrypoints/my-index.ts

```diff
@@ -32,6 +34,13 @@ class MyIndex extends LitElement {
 
   public connectedCallback() {
     super.connectedCallback();
+    if (isMobile && changeRequestedFromRedirect) {
```

Comment:
This can't happen, we hardcode the instance url to `homeassistant://navigate` when we're on mobile?

Reply:
The use case is when the link is opened in the app AFTER we got redirected. But I think we will need a different URL for this, to prevent the index to be opened in the app...